src/components/Offices/Identity/index.js

```diff
@@ -154,7 +183,7 @@ function IdentityTable({ info }) {
   return <Table columns={columns} data={data} />;
 }
 
-function IdentityInfo({ identity }) {
+function IdentityInfo({ identity, llmBalance }) {
```

Comment:
rename instances of llmBalance to centralizedLlmBalance or something, when referring to the data in the centralized database